test/distributed/fsdp/test_fsdp_checkpoint.py

```diff
@@ -166,13 +171,15 @@ def test_basic_checkpoint_end_to_end(self, cpu_offload, offload_activations):
             # Runs FSDP with no checkpointing
             fsdp_only_seq = FSDP(deepcopy(seq), cpu_offload=cpu_offload)
             # Runs checkpoint-wrapped FSDP
+            if offload_activations:
+                checkpoint_wrapper = offload_wrapper
```

Comment:
nit: maybe this should just be called `activation_wrapper` or `wrapper` (and change the instances below) or else you are shadowing the actual `checkpoint_wrapper` method, which might be confusing since it may be both `checkpoint_wrapper` or `offload_wrapper`